src/kernel.cr

```diff
@@ -1,18 +1,39 @@
 {% if flag?(:win32) %}
+  # The standard input file descriptor.
```

Comment:
"The standard input file descriptor" doesn't really add much more info than the name. I feel like more needs to be added here.

Reply:
Probably that it can contain output from piping and that keyboard input gets redirected to it?
```
Might contain piped output.
Keyboard input gets redirected here.
```